subsys/bluetooth/controller/ll_sw/ll_adv.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2017 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+struct ll_adv_set {
+	u8_t  chl_map:3;
+	u8_t  filter_policy:2;
+#if defined(CONFIG_BLUETOOTH_CONTROLLER_PRIVACY)
+	u8_t      rl_idx : 5;
+	u8_t      own_addr_type : 2;
+	u8_t      id_addr_type : 1;
+	u8_t      id_addr[BDADDR_SIZE];
```

Comment:
There's some inconsistency here in the alignment of the variable names, as well as with using a space before and after ':' which the other members don't have.